src/RJP.MultiUrlPicker/MultiUrlPickerValueConverter.cs

```diff
@@ -37,22 +37,24 @@ public override bool IsConverter(PublishedPropertyType propertyType)
 
         public override object ConvertDataToSource(PublishedPropertyType propertyType, object source, bool preview)
         {
-            if (string.IsNullOrWhiteSpace(source?.ToString()))
+            var sourceString = source?.ToString();
```

Comment:
Convert the source to a string once and then use it everywhere, no need to do the same over and over again 😀